assets/js/uno.js

```diff
@@ -0,0 +1,281 @@
+////////////////////////
+// Card Property
+///////////////////////
+
+// Color
+var Red = 0
+var Green = 1
+var Blue = 2
+var Yellow = 3
+// var cWild = 4
+
+// Value
+var Zero = 0
+var One = 1
+var Two = 2
+var Three = 3
+var Four = 4
+var Five = 5
+var Six = 6
+var Seven = 7
+var Eight = 8
+var Nine = 9
+var DrawTwo = 10
+var Skip = 11
+// var DrawFour = 12
+// var vWild = 13
+  // Removed Reverse Card since same as Skip
+
+// Card Image
+var cardURL = 'URL'

```

Comment:
you have multiple image url for your cards, so why only create one `string` variable?